internal/engine/buildcontroller_test.go

```diff
@@ -655,3 +655,27 @@ func TestBuildControllerUnresourcedYAMLFirst(t *testing.T) {
 	}
 	assert.Equal(t, expectedBuildOrder, observedBuildOrder)
 }
+
+func TestBuildControllerRespectDockerComposeOrder(t *testing.T) {
+	f := newTestFixture(t)
+	defer f.TearDown()
+
+	sancho := NewSanchoFastBuildDCManifest(f)
```

Comment:
can you make a note somewhere of what the order would be if we weren't respecting DC order and let the k8s-unbuilt logic handle this?

Reply:
comment added